src/main/java/com/google/sps/util/TestUtil.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.util;
+
+import com.google.appengine.api.datastore.Entity;
+import com.google.sps.data.Recipe;
+import java.util.ArrayList;
+
+public class TestUtil {
```

Comment:
Do you have future ideas for this class vs the TestData class? Since they're both small and do related things, it seems like you could combine them, unless you know that in the future it makes more sense for them to be separate.

Reply:
Yes, the idea is that as I begin working on more complex parts of search and want to make sure that my implementation is working and doesn't break, TestData will have static classes which represent a data set for a specific test. Each static class will have an initialize and clear data method for putting their data into the Datastore and deleting it during testing. The variables are public and static so they can be accessed from the test and values can be asserted using JUnit. 
Eventually, I hope to have one of these which can initialize data in the "real Datastore" for front end testing, and for the other nested classes have them take a parameter in the initialize and clear methods for a mock datastore reference which they will use.

TestUtil will have any functions that can be reused among the nested classes in TestData which help abstract functionality, but no actual Entity/recipe information.

Reply:
Sounds good, thanks for the explanation.